scraper_data.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 import click
 import json
-import data_scrapers
+from covid19_sfbayarea import data as data_scrapers
```

Comment:
I was trying to keep names from getting too painfully long/nested, but I may have made this one too generic. Happy to change if anyone has better ideas than `data`.

Reply:
`data` is clear enough to me. I thought about `county_data` but we will be plugging into sources other than county health departments.